### DISCUSSION TITLE
Mysterious new "Tabs from" parent tab

### DISCUSSION THREAD

**Discussion by mr-brunes** [piroor/treestyletab#2856] [General]
I've noticed that some mysterious new tab parents have appeared all labelled "Tabs from ...". On the far right I noticed that there is a link for "There is an option to deactivate opening of this type tab." but clicking on this just reveals a bewildering laundry list of options, most of which are incomprehensible to me. There is nothing that I can spot which says "Tab from" option. Nor is there anything in Release Notes. I'm sure many of these indecipherable options are incredibly useful but without a visual indicator of what they do, they will sadly remain a mystery.

**Comment by piroor** [comment]
It is created due to options TST Options => "New Tabs Behavior" => "Auto-grouping of tabs" => "Auto-group tabs opened from same pinned tab" or "Auto-group tabs opened from a bookmark folder".

When you click the "There is an option to deactivate opening of this type tab" link, the section "Auto-grouping of tabs" should be focused/highlighted, and the options page should be scrolled to the section automatically. Does the behavior not working on your environment?

**Reply by mr-brunes** [reply]
Tx for the explanation. It would help if this feature had a name e.g. "TST Auto-group child tabs from pinned parent" or somesuch that was clearly displayed on the autogroup tab page. Then the "There is an option .." link would be more obvious as to which option from which extension could be configured. It'd help too if it wasn't so far off to the right (easily overlooked).

**Reply by piroor** [reply]
Thank you for a feedback. Currently I have no best solution yet, but I've decided to add a description what will happen with the feature:
![image](https://user-images.githubusercontent.com/70062/115589908-5e7ab580-a30b-11eb-84a3-a97715b22c63.png)

**Reply by mr-brunes** [reply]
Tx for listening!

**Comment by pcider** [comment]
I have recently gotten this behaviour from pinned tabs (when clicking on two links from a pinned tab). It brings up this window. However, I couldn't find the setting to remove this mysterious tab.
![image](https://user-images.githubusercontent.com/28196972/236682089-80098238-4c49-4c82-a33c-336ec72a619a.png)
